Header header
uint8 robot_id
uint8 explored_sub_area